frontend/file_form.js

```diff
@@ -303,7 +303,13 @@ class UploadFile {
 
   handleDelete(uploadIndex) {
     const { url } = this.uploads[uploadIndex];
-
+    console.log(url);
+    if (url.endsWith('.placeholder')) {
```

Comment:
There is a chance that users will upload a file with `.placeholder` as suffix. I am not particularly worried about it though.

Reply:
Because the users can upload such a file successfully (without .placeholder attribute), and they can even delete it from the frontend, although the actual file will remain on the server for a day or two.